src/SimpleViewLibrary.php

```diff
@@ -398,7 +398,82 @@ public function updateHits($hitTypeID, $recId, $hitDate)
         } catch (Exception $e) {
             $results = false;
         }
-        
+
+        return $results;
+    }
+
+    /**
+     * Get listing that has been added since $lastSync date.
+     *
+     * Note: this is a custom method which only available for some clients.
+     *
+     * @param string $lastSync
+     *   Date string e.g. 2020-12-01
+     *
+     * @return array|bool array response or false if failed.
+     */
+    public function getNewListings($lastSync)
+    {
+        try {
+            $results = $this->_soapClient->getNewListings(
+                $this->_clientUserName,
+                $this->_clientPassword,
+                $lastSync
+            );
+        } catch (Exception $e) {
+            $results = false;
+        }
+
+        return $results;
+    }
+
+    /**
+     * Get listing that has been updated since $lastSync date.
+     *
+     * Note: this is a custom method which only available for some clients.
+     *
+     * @param string $lastSync
+     *   Date string e.g. 2020-12-01
+     *
+     * @return array|bool array response or false if failed.
+     */
+    public function getChangedListings($lastSync)
+    {
+        try {
+            $results = $this->_soapClient->getChangedListings(
```

Comment:
would be nice to abstract this in the future so not adding the same vars each time

```
public function apiGetResponse($name) {
  return $this->_soapClient->$name(
                $this->_clientUserName,
                $this->_clientPassword,
                $lastSync
            );
}
```

realise this is probably inherited from fork